client/state/sites/domains/assembler.js

```diff
@@ -18,6 +18,13 @@ function assembleGoogleAppsSubscription( googleAppsSubscription ) {
 }
 
 export const createSiteDomainObject = domain => {
+	let transferEndDate;
+
+	if ( domain.transferStartDate ) {
+		transferEndDate = new Date( domain.transferStartDate );
+		transferEndDate.setDate( transferEndDate.getDate() + 7 ); // Add 7 days.
+	}
```

Comment:
Does this work? The source `domain` object has a `transfer_start_date` property, not `transferStartDate`.

Slightly better way to write all this:
```js
let transferEndDate = null;
if ( domain.transfer_start_date ) {
  transferEndDate = new Date( domain.transfer_start_date );
  transferEndDate.setDate( ... );
  transferEndDate = transferEndDate.toIsoString();
}

return {
  ...
  transferEndDate, // already has the desired value, no calculation here
}
```

Reply:
Good point, done!